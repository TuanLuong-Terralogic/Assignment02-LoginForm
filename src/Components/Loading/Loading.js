import React from 'react';

const Loading = () => {
  return (
    <div data-testid="loading" className="loading"></div>
  );
};

export default Loading;