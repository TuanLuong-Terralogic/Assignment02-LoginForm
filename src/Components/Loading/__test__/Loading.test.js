import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Loading from '../Loading';

configure({ adapter: new Adapter() });

describe('Loading', () => {
  it('it should render', () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
    expect(component.find('div').props().className).toBe('loading')
  })
})