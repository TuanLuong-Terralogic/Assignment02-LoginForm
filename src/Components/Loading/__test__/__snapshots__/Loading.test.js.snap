// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading it should render 1`] = `ShallowWrapper {}`;
